### PR TITLE
Move clean

### PR DESCRIPTION
**Pull request by kenkehoe**
Moving clean from IO area to QC area for #159

### CODE REVIEW

**Comment by kenkehoe** [comment]
OK I'll look into the failure.

**Comment by AdamTheisen** [comment]
@kenkehoe I think you need to update the docs area to change the io.clean references.

https://github.com/ANL-DIGR/ACT/tree/master/docs/source/API

Looks like the following files.
Clean.rst:act.io.clean.CleanDataset
Clean.rst:.. autoclass:: act.io.clean.CleanDataset
io.rst:    Clean

File: docs/source/API/io.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.. automodule:: act.io
+.. automodule:: act.qc
```
Comment (AdamTheisen) [inline docs/source/API/io.rst]: Ken, i think in the io.rst file you need to change this qc back to IO and then remove the "Clean" line.

**Comment by AdamTheisen** [comment]
@kenkehoe I hope this is the last thing, but just as you pulled "Clean" out of the io.rst file, I think you need to put that in the qc.rst file

**Comment by kenkehoe** [comment]
Oh good catch @AdamTheisen . Thanks!

**Comment by AdamTheisen** [comment]
For some reason, my local build of docs is failing, but this looks to have passed.  I'll merge and see what happens.

**Comment by kenkehoe** [comment]
Awesome thanks!

**Pull request merged**